etc/helm/pachyderm/Chart.yaml

```diff
@@ -21,21 +21,21 @@ type: application
 # each time you make changes to the chart and its templates, including
 # the app version.  Versions are expected to follow Semantic
 # Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.3.0-nightly.1
```

Comment:
I don't think we want the nightly builds to be sequential integers starting from 1.  This should probably be 20210707 or similar.  (The trap here is that if you're incrementing a number, you need an atomic load-increment-store operation; consider the case where two nightly releases are running concurrently.  They will probably both do the same increment operation and overwrite each others' work.)

Another option may be: `2.3.0-nightly.20210707+(commit sha)` if we ever have two nightly builds on the same day.

See
https://pkg.go.dev/golang.org/x/mod/semver for the format available in the version number.

Reply:
yea agreed, that would be a better scheme for nightly. but apologies, developing story here, looks like we want to use `alpha` instead.